app/l10n/l10n.ts

```diff
@@ -0,0 +1,21 @@
+import { locale } from 'svelte-i18n-lingui';
+
+const reLang: RegExp = /[a-z]{0,2}/;
+
+export async function setLocale(lang: string) {
+  // TODO: Have a better fallback system that checks whether the locale
+  // is missing or if there's actually an error
+  // Support different variants of the same language i.e. en-US en-AU
+
+  // Extract only the 'en' from 'en-US'
+  lang = lang.match(reLang)[0];
+
+  try {
+    const { messages } = await import(`./locales/${lang}/messages.ts`);
```

Comment:
Can we please use the static `import` and not the dynamic `import()` ? The latter causes all kinds of hassles.

It's OK to load all languages.

Reply:
This may also fix the translation of the ts files.